### ISSUE TITLE
How to create pkl file of audio-image pairs and the whole scripts for complete pipeline

### ISSUE THREAD

**Issue by JackHenry1992** [shvdiwnkozbw/Multi-Source-Sound-Localization#5]
In your code, I see that you feed one image to the network. But AVE-dataset is composed of videos, do you only extract a single frame of a video? And I can not find the pkl file of audio-image pairs 
Furthermore, can you provide the detailed scripts for generate_vlabel and gt-data?

**Comment by shvdiwnkozbw** [comment]
For AVE dataset, we extract frames at 1fps, and use the image and its corresponding 1 second audio clip as a pair for training and evaluation. 
And the script for generating image pseudo labels is shown in generate_labelv.py
You can add the path of your pytorch pretrained resnet to the script for inference

**Comment by nakaotatsuya** [comment]
Hello, Thank you for creating generate_labelv.py .

for one video, you can have 10 audio/image pairs because each AVE-dataset video has 10 seconds, right?
And, I still don't know how to create the .pkl file of audio-image pairs.
I want to know the script for creating pkl file from raw datasets.

**Comment by poult-lab** [comment]
> Hello, Thank you for creating generate_labelv.py .
> 
> for one video, you can have 10 audio/image pairs because each AVE-dataset video has 10 seconds, right?
> And, I still don't know how to create the .pkl file of audio-image pairs.
> I want to know the script for creating pkl file from raw datasets.

I also want to know how to get audio/image pairs ...
